eb

```diff
@@ -126,9 +126,14 @@ export RSNT_EASYBUILD_MAGIC_COOKIE=263ca73bb634185aab1d1b41627fdbba
 
 # argument validation
 NEW_ARGS=()
-for argument in "$@"; do
-	if [[ "$CURRENT_USER" == "ebuser" ]]; then
-		if [[ $argument =~ --inject-checksums* || $argument =~ --robot* || "$argument" == "-r" ]]; then
+if [[ "$CURRENT_USER" == "ebuser" ]]; then
+	if ! [[ $(hostname) =~ ^login..archimedes.c3.ca$ ]]; then
+		echo "Please only submit as ebuser on login nodes"
```

Comment:
why restrict it to login node only ?

Reply:
do you want compute node sessions to send recursive jobs?

Reply:
I would say the following is valid workflow: 
1) salloc  to get a dedicated node
2) eb as myself to test and develop
3) sudo ebuser eb 

i.e. to use compute nodes as development nodes

Reply:
but for 3. should it spin off a job to a second compute node or stay on that node? Compile for both avx2 and avx512 or avx2 only?

Reply:
I would say it should behave just as if it was done on the login node, i.e. launch a new job and build for both avx2 and avx512